src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(rdfs:label obo:CL_4033027 "diffuse bipolar 1 cell")
+SubClassOf(obo:CL_4033027 obo:CL_0000750)
+
+# Class: obo:CL_4033028 (diffuse bipolar 2 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25309346") Annotation(oboInOwl:hasDbXref "PMID:27833534") obo:IAO_0000115 obo:CL_4033028 "An OFF diffuse bipolar cell that has preferential connectivity with ON parasol cells and bistratified amacrine cells. This cell contains a large number of synaptic ribbons and a small axon arbor area.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033028 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033028 "2023-04-04T10:49:07Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033028 "DB2 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033028 "A diffuse bipolar 2 cell makes an average of 133 ribbon synapses, 47 contacts are with OFF parasol cells and 4 contacts are with OFF midget cells.")
+AnnotationAssertion(rdfs:label obo:CL_4033028 "diffuse bipolar 2 cell")
+SubClassOf(obo:CL_4033028 obo:CL_0000750)
+
+# Class: obo:CL_4033029 (diffuse bipolar 3a cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033029 "An OFF calbindin-positive diffuse bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the Inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033029 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033029 "2023-04-04T10:53:33Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033029 "DB3a cell")
+AnnotationAssertion(rdfs:label obo:CL_4033029 "diffuse bipolar 3a cell")
+SubClassOf(obo:CL_4033029 obo:CL_0000750)
+
+# Class: obo:CL_4033030 (diffuse bipolar 3b cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033030 "An OFF calbindin-negative diffuse bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the Inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string.")
```

Comment:
Same [feedback as above](https://github.com/obophenotype/cell-ontology/pull/1922/files#r1158331964).

Reply:
"has a predominant connectivity" --> "predominantly connects to"